packages/atomic/src/components/atomic-icon/atomic-icon.tsx

```diff
@@ -79,18 +78,13 @@ export class AtomicIcon implements InitializableComponent {
   @Watch('icon')
   public async updateIcon() {
     const svgPromise = this.getIcon();
-    this.deferRenderingPromise = svgPromise;
     this.svg = await svgPromise;
   }
 
   public initialize() {
     this.updateIcon();
   }
 
-  public async componentWillRender() {
-    await this.deferRenderingPromise;
-  }
```

Comment:
removed because it's blocking rendering. The icon's size doesn't change and the background color is already visible for result icons. It doesn't seem like bad ux